docs/workflow-modules.qmd

```diff
@@ -52,8 +52,7 @@ You can use this function in combination with list comprehension to further redu
 
 A simple function cleans up your code, but still requires multiple calls to that function.
 
-```{python}
-#| source-line-numbers: "9-13"
```

Comment:
I noticed that this highlighting wasn't actually used because the `line-highlight` extension isn't enabled for this page, and when I added the extension, it highlighted what was probably the wrong set of lines.

But if line highlighting is desirable here, we could change change things to enable the extension and fix the highlighted lines.

This is what it currently looks like on the web site:

<img width="638" alt="image" src="https://github.com/posit-dev/py-shiny-site/assets/86978/8b595c1e-e3a5-472f-8d52-39bca9c9cb2b">

https://shiny.posit.co/py/docs/workflow-modules.html